src/views/esp/view_esp.cpp

```diff
@@ -71,6 +71,87 @@ namespace big
 			if (distance < g.esp.box_render_distance[1] && distance > g.esp.box_render_distance[0] && g.esp.box)
 				draw_list->AddRect({esp_x - (62.5f * multplr), esp_y - (175.f * multplr)}, {esp_x - (62.5f * multplr) + (125.f * multplr), esp_y - (175.f * multplr) + (350.f * multplr)}, esp_color);
 
+			if (distance < g.esp.skeleton_render_distance[1] && distance > g.esp.skeleton_render_distance[0] && g.esp.skeleton)
+			{
+				std::vector<PedBones> bone_ids = // size: 18
+				{
+				    PedBones::SKEL_Head,
+				    PedBones::SKEL_Neck_1,
+				    PedBones::SKEL_Spine_Root,
+				    PedBones::SKEL_Pelvis,
+				    PedBones::SKEL_L_Thigh,
+				    PedBones::SKEL_R_Thigh,
+				    PedBones::SKEL_L_Calf,
+				    PedBones::SKEL_R_Calf,
+				    PedBones::SKEL_L_Foot,
+				    PedBones::SKEL_R_Foot,
+				    PedBones::SKEL_L_Clavicle,
+				    PedBones::SKEL_R_Clavicle,
+				    PedBones::SKEL_L_UpperArm,
+				    PedBones::SKEL_R_UpperArm,
+				    PedBones::SKEL_L_Forearm,
+				    PedBones::SKEL_R_Forearm, 
+					PedBones::SKEL_L_Hand, 
+					PedBones::SKEL_R_Hand
+				};
+
+				std::vector<rage::fvector4> bones = {};
+				const size_t skel_size            = 18;
+
+				for (int i = 0; i < skel_size; i++)
+				{
+					rage::fvector4 out = {};
+					if (g_pointers->m_gta.m_get_ped_pone(plyr->get_ped(), out, bone_ids[i]))
```

Comment:
This function has a spelling mistake in it, and it was corrected in master, reflect it in your branch.

Reply:
fixed, and pushed a tiny improvement to cut down on repeated code, plus I think I missed to draw spine -> pelvis (not sure if spine and pelvis connect or not) so added that just incase